flow2ml/AutoResults.py

```diff
@@ -0,0 +1,106 @@
+from sklearn.metrics import roc_auc_score, roc_curve
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from sklearn.metrics import plot_confusion_matrix
+from sklearn.metrics import plot_precision_recall_curve
+from docx import Document
+from docx.shared import Inches
+
+class Auto_Results:
+    def __init__(self,model,X_test,Y_test):
+        self.model = model
```

Comment:
Please add documentation for every function that you defined here, so that anyone who see your code will understand easily.

Reply:
Okay